dfttopif/parsers/base.py

```diff
@@ -46,16 +51,38 @@ class DFTParser(object):
     _converged = None
     ''' Whether this calculation has converged '''
     
-    def __init__(self, directory):
-        '''Initialize a parser.
+    def __init__(self, directory, files=None):
+        '''Initialize a parser by defining the root directory of the calculation,
+            and a list of files the parser is allowed to access.
         
         Input:
-            directory - String, path to a directory of output files
+            directory - str, path to a directory of output files
+            files - [str], list of files usable by this parser. Each entry should be the complete path
```

Comment:
Are you proposing that this be supported in every parser type? If so, the pwscf parser needs to be updated.  If not, it shouldn't go in the base class.